routes/snippets.js

```diff
@@ -1,4 +1,42 @@
 const express = require("express");
 const router = express.Router();
 
+const mongoose = require("mongoose");
+const createError = require("http-errors");
+
+const Snippet = require("../models/Snippet");
+
+const {
+  OK,
+  INVALID_ID,
+  INVALID_REQUEST,
+  INTERNAL_SERVER_ERROR,
+} = require("../constants/messages");
```

Comment:
클라이언트쪽 파일에서 구조 분해 할당하는 형식에 맞춰 다음과 같이 작성하면 좋을 것 같습니다!

```suggestion
const MESSAGES = require("../constants/messages");

const {
  OK,
  INVALID_ID,
  INVALID_REQUEST,
  INTERNAL_SERVER_ERROR,
} = MESSAGES;
```